src/k8s/pkg/k8sd/app/hooks_start.go

```diff
@@ -24,6 +24,12 @@ func (a *App) onStart(ctx context.Context, s state.State) error {
 		}
 	}()
 
+	serverStatus, err := a.cluster.Status(ctx)
```

Comment:
TODO: this returns the host name instead of the node name that was passed to `NewCluster`.

https://chat.canonical.com/canonical/pl/b5c8nzioi3bazbn4k37g9q14xr

We'll need to fix that in microcluster or find another way of retrieving the node name.